src/kernels/kernelCommandListener.node.ts

```diff
@@ -235,42 +209,4 @@ export class KernelCommandListener implements IDataScienceCommandListener {
                 .ignoreErrors();
         }
     }
-    private onDidStartKernel(kernel: IKernel) {
-        this.kernelsStartedSuccessfully.add(kernel);
-    }
-    private onKernelStatusChanged({ kernel }: { status: KernelMessage.Status; kernel: IKernel }) {
```

Comment:
Remvoed code form here as:
* progress for restarting kernels has nothing to do with command handler
* DIsplaying warnings about dead kernels has nothing to do with command handler

As both of the above are two distinct features, i've moved into 2 separate files.